daemon/cmd/daemon.go

```diff
@@ -193,15 +193,11 @@ func (d *Daemon) init() error {
 		}
 
 		if option.Config.SockopsEnable {
-			disableSockops := func(err error) {
-				option.Config.SockopsEnable = false
-				log.WithError(err).Warn("Disabled '--sockops-enable' due to missing BPF kernel support")
-			}
 			eppolicymap.CreateEPPolicyMap()
 			if err := sockops.SockmapEnable(); err != nil {
-				disableSockops(err)
+				log.WithError(err).Error("Failed to enable Skmsg")
```

Comment:
The error message should be `Failed to enable Sockmap`

Reply:
Sure will fix it. It was originally Sockmap but based on previous review comment I have to change it to Skmsg, will revert it.